index.js

```diff
@@ -221,8 +221,17 @@ let moxios = {
   * @param {Function} fn The function to execute once waiting is over
   * @param {Number} delay How much time in milliseconds to wait
   */
-  wait: function(fn, delay = this.delay) {
-    setTimeout(fn, delay)
+  wait: function(...args) {
```

Comment:
Good idea. I didn't keep the type check before calling the callback because the variable is already guaranteed to be a function given the first line of the function.